WordPress/WordPressTest/ReaderReblogActionTests.swift

```diff
@@ -92,7 +79,9 @@ class ReblogPresenterTests: ReblogTestCase {
 
     func testPresentEditorForMultipleSites() {
         // Given
-        blogService!.blogCount = 2
+        for _ in 1...2 {
+            BlogBuilder(blogService!.managedObjectContext).with(visible: true).isHostedAtWPcom().withAnAccount().build()
+        }
```

Comment:
I'm a fan of using this approach, or it's equivalent `(1...2).forEach { ... }` to repeat actions. However, I wonder it it's overkill in this case, where we only have two instances?

Am I missing some advantage of using this loop over duplicating the line?

🤔 I guess having the loop make it explicit that the repetition is intentional. It also, obviously, removes duplication, making it easier to change the code in the future.

Anyway, I don't think this is wrong, just curious about your rationale for this and what you think about the tradeoffs. Happy for this to stay as is, mostly geeking out on little details.

Reply:
Yeah, the main reason was conveying the idea that the repetition is intentional, especially because a specific type of blog(with those three properties) is needed here.